lib/phoenix_html/tag.ex

```diff
@@ -48,10 +48,14 @@ defmodule Phoenix.HTML.Tag do
       ...> end
       {:safe, ["<p class=\"test\">", "Hello", "</p>"]}
   """
-  def content_tag(name, content) when is_atom(name) do
+  def content_tag(name, content) when is_atom(name) and not is_list(content) do
     content_tag(name, content, [])
   end
 
+  def content_tag(name, [do: block]) when is_atom(name) do
```

Comment:
Can you please move this clause up and ensure this still works:

    content_tag :p, ["hello", ?\s, "world"]

Thank you!